warehouse/integrations/deltalake-native/deltalake_native.go

```diff
@@ -0,0 +1,1356 @@
+package deltalake_native
+
+import (
+	"context"
+	"database/sql"
+	"database/sql/driver"
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	dbsqllog "github.com/databricks/databricks-sql-go/logger"
+
+	dbsql "github.com/databricks/databricks-sql-go"
+	"github.com/rudderlabs/rudder-server/warehouse/logfield"
+
+	"github.com/rudderlabs/rudder-go-kit/config"
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	"github.com/rudderlabs/rudder-go-kit/stats"
+	"github.com/rudderlabs/rudder-server/utils/misc"
+	warehouseclient "github.com/rudderlabs/rudder-server/warehouse/client"
+	"github.com/rudderlabs/rudder-server/warehouse/internal/model"
+	warehouseutils "github.com/rudderlabs/rudder-server/warehouse/utils"
+)
+
+const (
+	Host                   = "host"
+	Port                   = "port"
+	Path                   = "path"
+	Token                  = "token"
+	Catalog                = "catalog"
+	UseSTSTokens           = "useSTSTokens"
+	EnableExternalLocation = "enableExternalLocation"
+	ExternalLocation       = "externalLocation"
+	UserAgent              = "Rudderstack"
+)
+
+const (
+	provider       = warehouseutils.DELTALAKE
+	tableNameLimit = 127
```

Comment:
Can we please have a comment as to what this number represents and how we came up with `127`?

Reply:
Adding comment.